libs/server/Resp/AdminCommands.cs

```diff
@@ -270,6 +270,75 @@ private bool ProcessAdminCommands<TGarnetApi>(RespCommand command, ReadOnlySpan<
                     errorCmd = "ping";
                 }
             }
+            else if (command == RespCommand.HELLO)
+            {
+                int? protocolVersion = null;
+                ReadOnlySpan<byte> authUsername = default, authPassword = default;
+                string clientName = null;
+
+                if (count > 0)
+                {
+                    var ptr = recvBufferPtr + readHead;
+                    int _protocolVersion;
+                    if (!RespReadUtils.ReadIntWithLengthHeader(out _protocolVersion, ref ptr, recvBufferPtr + bytesRead))
+                        return false;
+                    readHead = (int)(ptr - recvBufferPtr);
+
+                    protocolVersion = _protocolVersion;
+                    count--;
+                    while (count > 0)
+                    {
+                        var param = GetCommand(bufSpan, out bool success1);
+                        if (!success1) return false;
+                        var paramStr = Encoding.ASCII.GetString(param);
+                        count--;
+                        if (paramStr.Equals("AUTH", StringComparison.OrdinalIgnoreCase))
```

Comment:
Can use CmdStrings.AUTH and then do a SequenceEquals instead of converting to ASCII & doing a string comparison

Reply:
I added a new `EqualsIgnoreCase` method - pls check carefully. And we can make a work item to convert other uses of CmdStrings to use this method so we can remove lowercase versions in CmdStrings.

Reply:
See `EqualsUpperCaseSpanIgnoringCase`